mscore/capxml.cpp

```diff
@@ -587,9 +612,9 @@ QList<BasicDrawObj*> Capella::readCapxDrawObjectArray(XmlReader& e)
                         const QStringRef& tag(e.name());
                         if (tag == "basic") {
                               // note: the <basic> element always follows the DrawObject it applies to
-                              if (bdo)
-                                    bdo->readCapx(e);
-                              else
+                              //if (bdo)
+                              //      bdo->readCapx(e);
+                              //else
```

Comment:
Better delete, we have git for obtaining previous content.

Reply:
Okei